packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -2406,6 +2406,86 @@ describe('shallow', () => {
       });
     });
 
+    describeIf(is('>= 16.3'), 'should not call componentWillReceiveProps after setState is called', () => {
```

Comment:
i changed this from `is('16.3.0-0')` because we want it to apply to v16.4 and later also.

Can you elaborate, though, on why this wouldn't be true for all versions of react?